src/display/metadata.js

```diff
@@ -79,40 +80,72 @@ class Metadata {
       });
   }
 
+  _getSequence(entry) {
+    const name = entry.nodeName;
+    if (name !== "rdf:bag" && name !== "rdf:seq" && name !== "rdf:alt") {
+      return null;
+    }
+
+    return entry.childNodes.filter(node => node.nodeName === "rdf:li");
+  }
+
+  _getCreators(entry) {
+    if (entry.nodeName !== "dc:creator") {
+      return false;
+    }
+    if (!entry.childNodes.length) {
+      return true;
+    }
+
+    // Child must be a Bag (unordered array) or a Seq.
+    const seqNode = entry.childNodes[0];
+    const authors = this._getSequence(seqNode) || [];
+    this._metadataMap.set(
+      entry.nodeName,
+      authors.map(node => node.textContent.trim())
+    );
+
+    return true;
+  }
+
   _parse(xmlDocument) {
     let rdf = xmlDocument.documentElement;
 
-    if (rdf.nodeName.toLowerCase() !== "rdf:rdf") {
+    if (rdf.nodeName !== "rdf:rdf") {
       // Wrapped in <xmpmeta>
       rdf = rdf.firstChild;
-      while (rdf && rdf.nodeName.toLowerCase() !== "rdf:rdf") {
+      while (rdf && rdf.nodeName !== "rdf:rdf") {
         rdf = rdf.nextSibling;
       }
     }
 
-    const nodeName = rdf ? rdf.nodeName.toLowerCase() : null;
+    const nodeName = rdf ? rdf.nodeName : null;
     if (!rdf || nodeName !== "rdf:rdf" || !rdf.hasChildNodes()) {
```

Comment:
With the lower-casing now happening in the XML-parser, you can inline the `nodeName` check here instead:
```suggestion
    if (!rdf || rdf.nodeName !== "rdf:rdf" || !rdf.hasChildNodes()) {
```